README.md

```diff
@@ -53,7 +56,8 @@ This file has to be valid JSON:
 }
 ```
 
-If you need to always read pages for different platform (e.g. Linux), you can put it in config file:
+If you need to always read pages for different platform (e.g. Linux),
+you can put it in config file:
```

Comment:
While we're at it.. "If you regularly need pages for a different platform" or something like that, "to always read" won't work.